src/intelligence_layer/core/model.py

```diff
@@ -195,13 +197,16 @@ def tokenize(self, text: str) -> Encoding:
 
 
 class ControlModel(ABC, AlephAlphaModel):
-    AllowedModel: Literal[""]
+    RecommendedModel: Literal[""]
```

Comment:
we currently no longer use the type hinting of this model, so we could just use a list instead of a literal, this would make the error check easier (no longer .__args__)